Hero section

ragimovalik opened the pull request:
Созданы файлы hero.html в partials и _hero.scss в новой папке layouts в scss. 
Внесено изменение в index.html, подключает файл секции героя через include.

ragimovalik committed:
Созданы файлы html+scss для секции Hero

Добавлена начальная разметка в секции Hero.
Мобильная версия

ragimovalik committed:
Добавлен фоновый цвет

ragimovalik committed:
Исправлен путь в <include>

ragimovalik commented:
Кстати, не применяются обнуление стилей для списка ul

mityaua commented:
А класс задан?

ragimovalik commented:
> А класс задан?

Моя невнимательность. я решил что это глобальный стиль для боди.

ragimovalik committed:
Добавлены оптимизированные изображения фонового круга для Hero

ragimovalik committed:
Merge branch 'develop' into hero

mityaua committed:
Merge branch 'develop' into hero

ragimovalik committed:
Обновление изображений в Hero (мобильная версия)

ragimovalik committed:
Merge branch 'hero' of https://github.com/mityaua/final-project-7 into hero

ragimovalik committed:
Черновые варианты позиционирования в секции Hero

ragimovalik committed:
Merge branch 'develop' into hero

mityaua commented:
Дублируется папка layouts. У нас уже есть layout. Думаю стоит туда перенести.

mityaua commented:
> layouts

После последнего восстановления сделал папку layouts, так что ничего не нужно переименовывать у себя.

ragimovalik committed:
Merge branch 'develop' into hero

ragimovalik committed:
Merge branch 'develop' into hero

ragimovalik committed:
Выполнена мобильная версия.

Разметка и стили для мобильной версии секции Hero.

mityaua commented:
Если не ошибаюсь. нужен еще бордер боттом снизу https://prnt.sc/uz6jrc

ragimovalik committed:
Merge branch 'develop' into hero

ragimovalik committed:
Merge branch 'develop' into hero

ragimovalik committed:
Удалены лишние изображения

ragimovalik committed:
Выполнена планшетная версия.

Выполнена планшетная версия.
1) разделение на 3 колонки с w:768рх.
2) В первую колонку добавлено изображение и ссылка на статью
3) Добавлена третья колонка с информацией о точках реализации (2 блока) и изображение.
4) Изображение в 3-ей колонке с абсолютным позиционированием.
5) Центрирование средней колонки начиная с 416рх.
5) Пересмотрены размеры контейнера и верхних отступов для колонок.
6) верстка с фиксированными размерами.
